policyengine_us/tests/policy/baseline/gov/aca/slspc_ca/slspc_age_adjusted_cost.yaml

```diff
@@ -0,0 +1,65 @@
+- name: slspc_age_adjusted_cost_person unit test 1 Alabama youth
+  period: 2023-01
+  input:
+    state_code_str: AL
+    monthly_age: 15
+    slspc_baseline_cost: 270
+  output:
+    slspc_age_adjusted_cost_person:
+      value: 270.00
+      absolute_error_margin: 0.1
+
+- name: slspc_age_adjusted_cost_person unit test 2 DC middle age
+  period: 2023-01
+  input:
+    state_code_str: DC
+    monthly_age: 55
+    slspc_baseline_cost: 388
+  output:
+    slspc_age_adjusted_cost_person:
+      value: 1028.12
+      absolute_error_margin: 0.1
+
+- name: slspc_age_adjusted_cost_person unit test 3 Utah young adult
+  period: 2023-01
+  input:
+    state_code_str: UT
+    monthly_age: 26
+    slspc_baseline_cost: 371
+  output:
+    slspc_age_adjusted_cost_person:
+      value: 637.67
+      absolute_error_margin: 0.1
+
+- name: slspc_age_adjusted_cost_person unit test 4 Minnesota middle age
+  period: 2023-01
+  input:
+    state_code_str: MN
+    monthly_age: 40
+    slspc_baseline_cost: 252
+  output:
+    slspc_age_adjusted_cost_person:
+      value: 361.87
+      absolute_error_margin: 0.1
+
+- name: slspc_age_adjusted_cost_person unit test 5 Mississippi adult
+  period: 2023-01
+  input:
+    state_code_str: MS
+    monthly_age: 33
+    slspc_baseline_cost: 240
+  output:
+    slspc_age_adjusted_cost_person:
+      value: 452.78
+      absolute_error_margin: 0.1
+
+- name: slspc_age_adjusted_cost_person unit test 6 West Virginia senior
+  period: 2023-01
+  input:
+    state_code_str: WV
+    monthly_age: 62
+    slspc_baseline_cost: 561
+  output:
+    slspc_age_adjusted_cost_person:
+      value: 2106.89
+      absolute_error_margin: 0.1
```

Comment:
```suggestion
- name: slspc_age_adjusted_cost_person unit test 6 West Virginia senior
  period: 2023-01
  absolute_error_margin: 0.1
  input:
    state_code_str: WV
    monthly_age: 62
    slspc_baseline_cost: 561
  output:
    slspc_age_adjusted_cost_person: 2_106.89
```

Reply:
all test files similar format

use "_" instead of ","
e.g. 5_444_000.01
3_933.1
33_112